Lib/idlelib/editor.py

```diff
@@ -1082,6 +1074,28 @@ def load_extension(self, name):
                 if hasattr(ins, methodname):
                     self.text.bind(vevent, getattr(ins, methodname))
 
+    def _load_extension_module(self, name):
+        fname = self.extfiles.get(name, name)
+        mod = None
+        first_exception = None
+        for modpath, package in [
+            ['.' + fname, __package__],
+            [fname, None],
+            ['.' + fname.lower(), __package__],
+            [fname.lower(), None],
+        ]:
```

Comment:
Given that the code you are expending worked fine, what is the point of this?  It apears to be extraneous to adding squeezer.

Reply:
If you email a response, please clip the quote. 

In 3.6, I made all idlelib file names lower case and broke ext filename == section name.
In compensation, I added a mapping
`extfiles = { # Map built-in config-extension section names to file names. ...`
before `def load_extension(self, name):` and a new first line:
`    fname = self.extfiles.get(name, name)`
that also works with added non-built-in extensions with filename == section-name,

This patch should add a line to the map:
`    'Squeezer': 'squeezer',`
and leave load_extension alone.

Reply:
This is required to avoid names the file `Squeezer` (note the capitalization). It won't be needed if we rework this to not use the extensions mechanism.

Reply:
Thanks for the clarification @terryjreedy!

I think this won't be needed anyways, but if we keep it as an extension I'll make the change you described.